packages/react/src/components/sideNavigation/mainLevel/MainLevel.module.scss

```diff
@@ -0,0 +1,48 @@
+.mainLevel {
+  composes: level from '../SideNavigation.module.scss';
+
+  font-size: var(--fontsize-body-m);
+
+  & > a,
+  & > button {
+    font-weight: 500;
+  }
+}
+
+.active {
+  composes: activeLevel from '../SideNavigation.module.scss';
+}
+
+.mainLevelListMenu {
+  display: none;
+}
+
+.mainLevelWithDivider {
+  composes: mainLevelWithDivider from '../SideNavigation.module.scss';
```

Comment:
The mainLevelWithDivider class definition seems to be only applied inside this file? Should we move the definition completely to here in order to decrease code indirection?

Reply:
- [ ] Remove unnecessary CSS indirections